From ceaedd2f63662333082940bfc5205d4db0a87e62 Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Wed, 1 Sep 2021 16:27:36 -0700
Subject: [PATCH 137/196] Remove spurious print in `abc_rpc_avalancheproof`
 (#220)

This little guy got errorenously merged when backporting v0.23.6.
Delete it.
---
 test/functional/abc_rpc_avalancheproof.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/test/functional/abc_rpc_avalancheproof.py b/test/functional/abc_rpc_avalancheproof.py
index ae8f987d3..b79743795 100644
--- a/test/functional/abc_rpc_avalancheproof.py
+++ b/test/functional/abc_rpc_avalancheproof.py
@@ -318,7 +318,6 @@ class AvalancheProofTest(BitcoinTestFramework):
                 ["amount"] - Decimal('0.01')}
         )
         signed_tx = node.signrawtransactionwithkey(raw_tx, [addrkey0.key])
-        print(signed_tx["hex"])
         node.sendrawtransaction(signed_tx["hex"])
         node.generate(1)
         wait_until(lambda: proofid not in get_proof_ids(node))
-- 
2.39.5 (Apple Git-154)

